coffeecard/CoffeeCard.WebApi/Pages/VerifyEmail.cshtml

```diff
@@ -0,0 +1,20 @@
+﻿@page
+@model CoffeeCard.WebApi.Pages.VerifyEmail
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@{
+    ViewData["Title"] = "Verify your email address";
+    Layout = "Shared/_Layout";
+}
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <title></title>
+</head>
+<body>
+<div>
+    
+</div>
+</body>
+</html>
```

Comment:
These are empty, are they needed?

Reply:
Probably not, since everything is happening in the model. I can try removing it if you consider it an eyesore